.github/workflows/build.yml

```diff
@@ -103,13 +103,47 @@ jobs:
           path: ./TestResults/**/*.trx
           if-no-files-found: error
 
+      - name: Add msbuild to PATH
+        if: runner.os == 'Windows'
+        uses: microsoft/setup-msbuild@v1.1
+
+      - name: Build BicepInVisualStudio.sln
```

Comment:
What kind of impact do these tests have on build duration?  As long it's not much now it's fine.  Otherwise in future might need to look into making sure it's spawned to a separate job.

Reply:
The newly added tests take about 2 mins on my machine and more than 10 mins on the CI. So moved the build, install and test steps I added to a new job so that it can happen in parallel. Made changes in multiple commits, so not adding here. Let me know if you have any questions/concerns.